bigbluebutton-tests/playwright/core/page.js

```diff
@@ -135,7 +135,7 @@ class Page {
   }
 
   async closeAudioModal() {
-    await this.waitForSelector(e.audioModal, ELEMENT_WAIT_LONGER_TIME);
+    await this.waitForSelector(e.audioModal, 15000);
```

Comment:
let's avoid using static values in cases like this. If needed, use `ELEMENT_WAIT_EXTRA_LONG_TIME` instead